c7n/resources/glue.py

```diff
@@ -185,6 +186,62 @@ class resource_type(TypeInfo):
     augment = universal_augment
 
 
+class SecurityConfigFilter(RelatedResourceFilter):
```

Comment:
Thanks @egunnar, way to implement the RelatedResourceFilter for this. I understand that this enables identifying Glue resources without RelatedSecurityConfigs and supports reporting on what configs each one has etc. @kapilt, what do you think? Does this hit the mark?

Reply:
yup this looks good